client/src/modules/email-report/modal/categories-people-list-modal.html

```diff
@@ -0,0 +1,50 @@
+<!-- billing service form -->
+<form
+    name="BillingServicesForm"
+    bh-submit="BillingServicesFormCtrl.submit(BillingServicesForm)"
+    novalidate>
+
+    <div class="modal-header">
+        <ol class="headercrumb">
+            <li class="static">
+                {{"FORM.LABELS.PROFILE_FROM" | translate}} 
+                {{catPeopleListCtrl.selectedCategory.table_display | translate}}
+            </li>
+        </ol>
+    </div>
+
+    <div class="modal-body" style="height: 100%;border: none !important">
```

Comment:
Why do you need to put `height:100%`?  Do these `style` tags actually do anything?